src/main/java/de/bayern/gdi/gui/Controller.java

```diff
@@ -445,53 +479,81 @@ private void extractBoundingBox() {
             bbox += envelope.getY() + ",";
             bbox += (envelope.getX() + envelope.getWidth()) + ",";
             bbox += (envelope.getY() + envelope.getHeight());
-            this.dataBean.addAttribute("bbox", bbox);
+            this.dataBean.addAttribute("bbox", bbox, "");
         } else {
             // Raise an error?
         }
     }
 
+
+    private boolean validateInput() {
+        String failed = "";
+        ArrayList<DataBean.Attribute> attributes
+                                = this.dataBean.getAttributes();
+        boolean ret = true;
```

Comment:
ret is set to false every time a attribute is not valid. The failed string is none empty in this case, too (Assuming attribute is none empty). So a none-empty failed string is equivalent to an explicit false set ret value.